river/tree/losses.py

```diff
@@ -0,0 +1,52 @@
+import math
+from abc import ABCMeta
+
+from .utils import GradHess
+
+
+class Loss(metaclass=ABCMeta):
```

Comment:
I think inheriting from `abc.ABC` is the more modern way, but not sure.

Other question: is there a reason why you can't use the `Loss` class from `optim.losses`?

Reply:
Yeap! You're right. I'll change that.

> Other question: is there a reason why you can't use the `Loss` class from `optim.losses`?

In the previous PR, I had [considered](https://github.com/online-ml/river/pull/458#issuecomment-785095468) this possibility.

But then we reached [another conclusion](https://github.com/online-ml/river/pull/458#issuecomment-798211842).

Reply:
Ok perfect 👌, I forgot 👴